.github/workflows/codeql-analysis.yml

```diff
@@ -2,9 +2,9 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ main, v6 ]
+    branches: [ main ]
```

Comment:
Do we not need to add v5 here now? We will still need to make v5 releases

Reply:
we will do that in the v5 branch